.lgtm.yml

```diff
@@ -4,9 +4,9 @@ extraction:
       command:
         - mkdir _lgtm_build_dir
         - cd _lgtm_build_dir
-        - wget -O apache-geode.zip http://mirror.transip.net/apache/geode/1.7.0/apache-geode-1.7.0.zip
-        - unzip apache-geode.zip
-        - cmake -DGEODE_ROOT="`pwd`/apache-geode-1.7.0" ..
+        - wget -O apache-geode.tgz http://mirror.transip.net/apache/geode/1.9.0/apache-geode-1.9.0.tgz
+        - tar xzf apache-geode.tgz
+        - cmake -DCMAKE_INSTALL_PREFIX="`pwd`/apache-geode-native-install" -DGEODE_ROOT="`pwd`/apache-geode-1.9.0" ..
```

Comment:
We don't need to set the install directory, especially if we aren't installing it.

Reply:
I agree. It was something I added to try fix the problem with the linker with LGTM but obviously did not help.